distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
I am fine with switching the javadoc plugin off for normal compiles, and only having it active in the release profile. If users want to generate the javadoc out of the git without releasing, they can manually run mvn javadoc:jar